django_cypress/views.py

```diff
@@ -1,11 +1,44 @@
 import json
+from http import HTTPStatus
 
+from django.contrib.auth import get_user_model
 from django.core import management
 from django.http import HttpRequest, JsonResponse
 from django.middleware.csrf import get_token
 from django.views import View
 
 
+class CreateUserView(View):
+    """A view for creating a user via HTTP POST requests."""
+
+    def post(
+        self,
+        request: HttpRequest,
+    ) -> JsonResponse:
+        """Handle HTTP POST requests to create a user.
+
+        Args:
+        ----
+        request (HttpRequest): The HTTP request object containing the user data.
+
+        Returns:
+        -------
+        JsonResponse: A JSON response indicating the success of the user creation.
+        """
+        body = json.loads(request.body.decode("utf-8"))
+
+        try:
+            user_model = get_user_model()
+            user = user_model.objects.create_user(**body)
+            return JsonResponse(
+                {"success": True, "user_id": user.id}, status=HTTPStatus.CREATED
```

Comment:
@Stavrospanakakis, since we are using status codes like `HTTPStatus.CREATED` and `HTTPStatus.BAD_REQUEST`, do we need to add the `success` attribute in the response? It feels redundant. What do you think?

Reply:
Correct, fixed it :smile: